.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: Continuous Integration
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  unit-test:
+    runs-on: ubuntu-latest
+    container:
+      image: circleci/node:12-browsers
+    env:
+      NPM_CONFIG_UNSAFE_PERM: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Fix ssh+git access
+        run: git config --global url."https://github.com/".insteadOf "git@github.com:"
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
+      - run: npm install
+      - run: npm run build:prod
+      - name: Unit test
+        run: npm run test
+
+  local-integration-tests:
+    runs-on: ubuntu-latest
+    container:
+      image: circleci/node:12-browsers
+    env:
+      NPM_CONFIG_UNSAFE_PERM: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      # - name: Fix ssh+git access
+      #   run: git config --global url."https://github.com/".insteadOf "git@github.com:"
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
+      - run: npm install
+      - run: npm run build:release
```

Comment:
If this works out, add lint:markdown as a dependency for build:release since I didn't think of that.